pkg/api/apiloader.go

```diff
@@ -104,6 +111,10 @@ func (a *Apiloader) LoadContainerService(contents []byte, version string, valida
 // SerializeContainerService takes an unversioned container service and returns the bytes
 func (a *Apiloader) SerializeContainerService(containerService *ContainerService, version string) ([]byte, error) {
 	switch version {
+	case v20170831.APIVersion:
+		// TODO: implement this...
```

Comment:
@brendanburns  @amanohar I assume this PR has value with this unimplemented? (and the resultant conversions that will ostensibly go in `converterfromapi.go`)